### DISCUSSION TITLE
[x]it! for IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by coutvv** [jotaen/xit#62] [Show & tell]
Hi! I really like this format, so I have created [the IntelliJ IDEA plugin](https://plugins.jetbrains.com/plugin/23646-xit-support) for support it in IDEA.

I would like you to add it to the Editor Support list on the [site](https://xit.jotaen.net/).

**Comment by jotaen** [comment]
Oh cool, thanks for creating this plugin! I’ve just added the link to the website 🎉
